DataFormats/interface/Electron.h

```diff
@@ -87,6 +92,7 @@ namespace cat {
     
     int snuID_;
     bool isTrigMVAValid_;
+    bool isHLTEle32WPTight_;
```

Comment:
Adding a member variable just for one era is not a good idea in general, you will loose backward / forward compatibility.

Reply:
I and T.J. discussed about this issue.
First of all, this seems easiest way to check if trigger path (HLT_Ele32_WPTight_L1Double) passed singleEGOr L1 filter.
Secondly, people are recommended to use proper cattools and cmssw version to open and analysis catTuples. Even if the content of this var & function change in future, it would be okay.

Also, T.J. suggested that we may change the name of var and function to general one (let's say, 'isTrgPass' or something like that), so we can put some useful information on future production (in case of 2018 or later data).